samples/ExampleGallery/Effects/DynamicBlurVideoEffect.cs

```diff
@@ -0,0 +1,86 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+//
+// Licensed under the MIT License. See LICENSE.txt in the project root for license information.
+
+using System.Collections.Generic;
+using Windows.Foundation.Collections;
+using Windows.Graphics.DirectX.Direct3D11;
+using Windows.Media.Effects;
+using Windows.Media.MediaProperties;
+using Microsoft.Graphics.Canvas;
+using Microsoft.Graphics.Canvas.Effects;
+
+namespace ExampleGallery.Effects
+{
+    /// <summary>
+    /// Win2D Gaussian Blur - http://microsoft.github.io/Win2D/html/T_Microsoft_Graphics_Canvas_Effects_GaussianBlurEffect.htm
+    /// BlurAmount (in DIP) - 0 is no blur, default is 3, max for this demo is 12
+    /// </summary>
+    public sealed class DynamicBlurVideoEffect : IBasicVideoEffect
+    {
+        private CanvasDevice _canvasDevice;
+        private IPropertySet _configuration;
+
+        /// <summary>
+        /// Value used for BlurAmount property
+        /// </summary>
+        public double BlurAmount
+        {
+            get
+            {
+                if (_configuration != null && _configuration.ContainsKey("BlurAmount"))
+                    return (double)_configuration["BlurAmount"];
+
+                return 3;
+            }
+            set
```

Comment:
The Setter for this property is never used - and in fact probably is misguiding since there's no way that the app can use this to set the property.

Suggest removing it.